rules/Renaming/Rector/Name/RenameClassRector.php

```diff
@@ -94,14 +94,16 @@ public function getNodeTypes(): array
     /**
      * @param FunctionLike|Name|ClassLike|Expression|Namespace_|Property $node
      */
-    public function refactorWithScope(Node $node, Scope $scope): ?Node
+    public function refactor(Node $node): ?Node
     {
         $oldToNewClasses = $this->renamedClassesDataCollector->getOldToNewClasses();
         if ($oldToNewClasses !== []) {
+            $scope = $node->getAttribute(AttributeKey::SCOPE);
```

Comment:
I still find why `Scope` for RenameClassRector after refactor require `ORIGINAL_NODE` instead of Node itself at 

https://github.com/rectorphp/rector-src/blob/3fa4bec792258ae18bf56c665d5ec3fa4938d475/src/Rector/AbstractRector.php#L342-L343

that's possibly due to docblock is too late to get `Scope`, that's another PR to apply if possible ;)